rpm/kvm/qm-kvm.spec

```diff
@@ -4,25 +4,15 @@
 %global rootfs_qm %{_prefix}/lib/qm/rootfs/
 
 Name: qm-kvm
-Version: 0
+Version: 0.7.3
 Release: 1%{?dist}
 Summary: Drop-in configuration for QM containers to mount bind /dev/kvm
 License: GPL-2.0-only
 URL: https://github.com/containers/qm
 Source0: %{url}/archive/qm-kvm-%{version}.tar.gz
 BuildArch: noarch
 
-# Build requirements
-BuildRequires: make
-BuildRequires: podman
-BuildRequires: selinux-policy
-BuildRequires: selinux-policy-devel
-
-# Runtime dependencies
 Requires: qm = %{version}-%{release}
```

Comment:
@dougsland this is tricky
@raballew tried to install it on AutoSD and had dependency issues, I do not think it is a must, 
Can we dismiss this dependency?

Reply:
As we've discovered, this dependency is too strict to be practical.  `%{release}` includes `%{dist}` defined by the system, which might differ from `%{dist}` defined by the buildsystem used to build qm.  Furthermore, this potentially prevents qm from being updated as it would result in broken dependencies.

Relax this to `Requires: qm >= %{version}` to avoid these... challenges.

Reply:
Ack

Reply:
>= works?

Reply:
or simply remove for now to unlock @raballew

Reply:
just checking